python/tvm/relay/op/_transform.py

```diff
@@ -590,18 +593,21 @@ def squeeze_shape_func(attrs, inputs, _):
     """
     axis = attrs.axis if attrs.axis is None else get_const_tuple(attrs.axis)
     keep_axes = []
+    remove_axes = []
```

Comment:
Isn't `remove_axes` the same of `axis`?

Reply:
The value in remove_axes will be limited by inputs[0].shape